contracts/src/BackedCommunityTokenDescriptorV1.sol

```diff
@@ -3,20 +3,318 @@ pragma solidity 0.8.12;
 
 import {IBackedCommunityTokenDescriptorV1} from "./interfaces/IBackedCommunityTokenDescriptorV1.sol";
 
+import {IBackedBunnyTraitRenderer} from "./traits/IBackedBunnyTraitRenderer.sol";
+import "./utils/Strings.sol";
+
 contract BackedCommunityTokenDescriptorV1 is IBackedCommunityTokenDescriptorV1 {
     uint256 constant activityCategoryId = 0;
-    uint256 constant contributorCateogryId = 1;
+    uint256 constant contributorCategoryId = 1;
     uint256 constant communityCategoryId = 2;
-    uint256 constant communityCallsCategoryId = 3;
-    uint256 constant loansReceivedCateogryId = 4;
-    uint256 constant loansGivenCateogryId = 5;
-    uint256 constant pullRequestsCategoryId = 6;
 
     function tokenURI(
```

Comment:
I would say this is a misnomer. Should be named like "svgImage" or something? It's not the tokenURI as that would be a json response?